docs/installation.md

```diff
@@ -1,9 +1,11 @@
-# Clone the repository
+# Installation and Configuration for cfgov-refresh
+
+## Clone the repository
 
 Using the console, navigate to the root directory in which your projects live and clone this project's repository:
 
 ```bash
-git clone git@github.com:cfpb/cfgov-refresh.git
+git clone https://github.com/cfpb/cfgov-refresh.git
```

Comment:
For consistency across the GitHub universe... in my experience `git clone https:://...` is the norm. I don't have major preference for this though, but I think the specific problem you're referencing isn't a good reason not to recommend it. I find that https works perfectly fine on our local machines.

https://help.github.com/articles/caching-your-github-password-in-git/